tt-train/sources/ttml/CMakeLists.txt

```
@@ -0,0 +1,111 @@
+project(ttml)
+
+file(
+    GLOB_RECURSE SOURCES
+    LIST_DIRECTORIES true
+    *.hpp
+    *.cpp
+) # I am a bad person
+
+add_library(ttml STATIC ${SOURCES})
+
+target_include_directories(
+    ttml
+    SYSTEM
+    PUBLIC
+        # Metalium
+        $ENV{TT_METAL_HOME}
+        $ENV{TT_METAL_HOME}/tt_metal
+        $ENV{TT_METAL_HOME}/tt_metal/third_party/umd
+        $ENV{TT_METAL_HOME}/tt_metal/hw/inc/wormhole
+        $ENV{TT_METAL_HOME}/tt_metal/hw/inc/wormhole/wormhole_b0_defines
+        $ENV{TT_METAL_HOME}/tt_metal/hw/inc/
+        $ENV{TT_METAL_HOME}/tt_metal/third_party/umd/src/firmware/riscv/wormhole
+        $ENV{TT_METAL_HOME}/tt_metal/third_party/umd/device
```

Should all be replaced with:

```suggestion
         Metalium::Metal
```
in `target_link_libraries`.

If anything fails, then that's a failing of Metal and should be fixed in `tt_metal`, not worked around in here.

---

Done